packages/components/popover/src/PopoverClose.tsx

```diff
@@ -0,0 +1,42 @@
+import * as RadixPopover from '@radix-ui/react-popover'
+import { Icon } from '@spark-ui/icon'
+import { IconButton } from '@spark-ui/icon-button'
+import { Close as CloseSVG } from '@spark-ui/icons/dist/icons/Close'
+import { cx } from 'class-variance-authority'
+import { forwardRef, useEffect } from 'react'
+
+import { usePopover } from './PopoverContext'
+
+export type CloseProps = RadixPopover.PopoverCloseProps & {
+  'aria-label': string
+}
+
+export const Close = forwardRef<HTMLButtonElement, CloseProps>(
```

Comment:
IMO this component should be named `CloseIconButton`

Reply:
I wanted to stick with Radix naming.

Reply:
I think that is better to name it like that some developers know that is an `IconButton`

Reply:
Why do they need to know it is an `IconButton` that we chose to use ? I think it may be too descriptive.

`Popover.CloseButton` maybe ? Its true that `Close` as a name might be too short and can be mistaken for an action/verb.

- Most design have no close button for Popovers
- The one who does either name it `CloseButton` (ex: Chakra) or `Close` (Radix)